database/scripts/0001-create-initial-tables.sql

```diff
@@ -51,4 +51,24 @@ CREATE TABLE public.event_rating
     event_id INTEGER NOT NULL REFERENCES public.events (id),
 
     CONSTRAINT unique_user_per_event UNIQUE (created_by, event_id)
-);
\ No newline at end of file
+);
+
+CREATE TABLE public.user_settings
+(
+    id SERIAL PRIMARY KEY,
+    receive_top_rated BOOLEAN NOT NULL DEFAULT false,
+
+    user_id INTEGER NOT NULL REFERENCES public.users (id),
+
+    CONSTRAINT unique_user UNIQUE (user_id)
+);
+
+CREATE TABLE public.sport_subscriptions
```

Comment:
Would maybe `user_sport_subscriptions` be a better name, to better indicate that it belongs to user?

Reply:
Agreed, changed in https://github.com/mbrizic/ratethegame/pull/28/commits/bf81d50858dd1dd5210749a30efe2620fd3a3534